src/coreclr/jit/lsra.cpp

```diff
@@ -4590,27 +4542,42 @@ RegRecord* LinearScan::getSecondHalfRegRec(RegRecord* regRec)
 //    A RegRecord which forms same double register with regRec
 //
 RegRecord* LinearScan::findAnotherHalfRegRec(RegRecord* regRec)
+{
+    regNumber anotherHalfRegNum = findAnotherHalfRegRec(regRec->regNum);
+    return getRegisterRecord(anotherHalfRegNum);
+}
+//------------------------------------------------------------------------------------------
+// findAnotherHalfRegRec: Find another half RegRecord which forms same ARM32 double register
+//
+// Arguments:
+//    regNumber - A float regNumber
+//
+// Assumptions:
+//    None
+//
+// Return Value:
+//    A RegRecord which forms same double register with regRec
+//
+regNumber LinearScan::findAnotherHalfRegRec(regNumber regNum)
```

Comment:
This function should be renamed `findDoubleRegPair` or similar: it shouldn't imply that it's finding a RegRecord.

Reply:
Sure. I have changed it to `findAnotherHalfRegNum()`